bindings/protocols/profinet/index.html

```diff
@@ -0,0 +1,1040 @@
+            <td>optional</td>
+            <td><a href="#payloaddatatype">PayloadDataType</a></td>
+
+        <tr>
+        
+                </tbody>
+            </table>
+
+        </section>
+        <section id="enumeratedValue">
+            <h3>Enumerated Value</h3>
+            <p>
+                The enumeratedValue class within the PROFINET binding template helps provide context to encoded parameters in the payload. Bit payloads in PROFINET are sometimes
+                used as encoded payload to logical mean something aside from 1 and 0. Informaiton about how the encoding payload will be decoded is defined in this section. 
+                The table below defines the terms used in the enumeratedValue class.
+            </p>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <!-- Generate forms terms from the ontology-->
+
+        <tr>
+            <td><code>profv:encodedPayload</code></td>
+            <td>used for <code>object</code> type properties to identify the bit mask of a proterty from the byte stream or byte payload</td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#number">number</a> or <a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
+
+        <tr>
+
+        <tr>
+            <td><code>profv:decodedPayload</code></td>
+            <td>used for <code>object</code> type properties to identify the bit mask of a proterty from the byte stream or byte payload</td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#string">string</a> , <a href="http://www.w3.org/2001/XMLSchema#number">number</a> or <a href="http://www.w3.org/2001/XMLSchema#boolean">boolean</a></td>
+
+        <tr>
+        
+        
+                </tbody>
+            </table>
+
+        </section>
+    </section>
+    <section id="mappings">
+        <h2>Mappings</h2>
+        <p>This section describes strategies and default values to employ protocol specific concepts within the <a href="https://w3c.github.io/wot-architecture/#dfn-interaction-model">WoT Interaction
+        model</a>. </p>
+        <section id="default-mappings">
+            <h3>Default mappings</h3>
+            <p>
+                The following table lists the default mappings between the protocol specific concepts and the WoT concepts. Please note that operations that are not
+                listed in the table are not supported by this binding template. For example, since the scope of this binding focuses on acyclic communication of the PROFINET protocol, the
+                <code>subscribeevent</code> operation is not supported.
```

Comment:
It would be good to give a full overview all of InteractionAffordances here:

- PropertyAffordance: readproperty, writeproperty supported. are observeproperty, unobserveproperty also supported?
- ActionAffordance: not supported at all?
- EventAffordance: not supported at all? (Can an IO Supervisor receive alarms?)